cwatm/hydrological_modules/lakes_reservoirs.py

```diff
@@ -293,7 +294,32 @@ def buffer_waterbody(rec):
                                               np.where(self.var.waterBodyTypC == 3, 1, self.var.waterBodyTypC))
 
             np.put(self.var.waterBodyTyp, self.var.decompress_LR, self.var.waterBodyTypC)
-
+            
+            # needs to be changed - maybe update all reservoir to spreadsheet
+            self.var.resId_restricted = globals.inZero.copy()
+            resIdList = np.unique(list(self.var.wastewater_to_reservoirs.values()))
```

Comment:
self.var.wastewater_to_reservoirs might not exist

Reply:
Included an 'if self.var.includeWastewater:' statement